targets/esp8266/source/jerry_extapi.c

```diff
@@ -16,7 +16,7 @@
 #include <stdlib.h>
 #include <stdio.h>
 
-#include "jerry-core/jerryscript.h"
+#include "jerry-core/include/jerryscript.h"
```

Comment:
This is so deep. What about creating a jerry-include top level directory and put the public api there.